lib/internal/Magento/Framework/GraphQl/Schema/Type/ScalarTypes.php

```diff
@@ -13,23 +13,27 @@
 class ScalarTypes
 {
     /**
+     * Check if type is scalar
+     *
      * @param string $typeName
      * @return bool
      */
     public function isScalarType(string $typeName) : bool
     {
-        $internalTypes = \GraphQL\Type\Definition\Type::getInternalTypes();
+        $internalTypes = \GraphQL\Type\Definition\Type::getStandardTypes();
         return isset($internalTypes[$typeName]) ? true : false;
     }
 
     /**
+     * Get instance of scalar type
+     *
      * @param string $typeName
      * @return \GraphQL\Type\Definition\ScalarType|\GraphQL\Type\Definition\Type
      * @throws \LogicException
      */
     public function getScalarTypeInstance(string $typeName) : \GraphQL\Type\Definition\Type
     {
-        $internalTypes = \GraphQL\Type\Definition\Type::getInternalTypes();
+        $internalTypes = \GraphQL\Type\Definition\Type::getStandardTypes();
```

Comment:
Variable should be renamed accordingly.
Is "getInternalTypes" deprecated in the new version of webonyx?

Reply:
I'll update. Yes, it was deprecated https://github.com/webonyx/graphql-php/blob/e804ccf73add32be937d392abb8b359243817d57/src/Type/Definition/Type.php#L189-L199